kcp/operator.yaml

```diff
@@ -0,0 +1,438 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: controller-manager
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  labels:
+    control-plane: argocd-operator
+spec:
+  selector:
+    matchLabels:
+      control-plane: argocd-operator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: argocd-operator
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      ## Disable auto-mounting SA token 
+      automountServiceAccountToken: false
+      containers:
+      - command:
+        - /manager
+        # Disabled leader election arg --leader-elect
+        image: quay.io/cbanavik/gitops-operator:v1.6
```

Comment:
Here, I'm using a custom operator image that includes the workarounds in the argocd-operator 

https://github.com/argoproj-labs/argocd-operator/compare/master...chetan-rns:argocd-operator:argocd-operator-kcp?expand=1